services/jwt.go

```diff
@@ -180,7 +180,11 @@ func (j *JWTService) parseToken(tokenString string, claimKey string) (uuid.UUID,
 		var audStr string
 		audRaw, exists := claims[audClaim]
 		if exists {
-			audStr = audRaw.(string)
+			audVal, ok := audRaw.(string)
```

Comment:
`ok` is already declared, so I think this could be simplified to `audStr, ok = audRaw.(string)`

Reply:
Alternatively, we could call `claims.GetAudience()` which handles the parsing and type checking